src/ts/components/settingspanel.ts

```diff
@@ -113,6 +115,15 @@ export class SettingsPanel extends Container<SettingsPanelConfig> {
       });
     }
 
+    if (config.pageTransitionAnimation) {
+      const handleResize = () => {
+        // Reset the dimension of the settingsPanel to let the browser calculate the new dimension after resizing
+        this.getDomElement().css({ width: '', height: '' });
+      };
+
+      player.on(player.exports.PlayerEvent.PlayerResized, handleResize);
+    }
```

Comment:
There is a workaround within the animation handling where the width and height of the `SettingsPanel` are set to the current dimension. This breaks the `SettingsPanel` when resizing the browser window. Removing the width and height properties on resize fixes this.